nuitka/build/SingleExe.scons

```diff
@@ -138,9 +138,9 @@ uninstalled_python = getBoolOption("uninstalled_python", False)
 # Unstriped mode: Do not remove debug symbols.
 unstripped_mode = getBoolOption("unstripped_mode", False)
 
-# Clang compiler mode, forced on macOS X and FreeBSD, optional on Linux.
```

Comment:
I recently got told about macOS being the proper spelling.

Reply:
You're right! I did not notice that when I copy/paste it.